action.yml

```diff
@@ -22,19 +22,21 @@ inputs:
 runs:
   using: "composite"
   steps:
-    # Set bot-email input to dynamic default value, if needed
-    - name: "Construct bot-email from inputs.bot-name if inputs.bot-email not provided"
-      id: bot-email-set
+    # Git config
+    - name: "Git config"
       shell: bash
       run: |
+        #Construct bot-email from inputs.bot-name if inputs.bot-email not provided
+        bot_id=$(curl -sSL 'https://api.github.com/users/${{ inputs.bot-name }}' | jq .id)
         if [ -z ${{ inputs.bot-email }} ]; then
-          bot_id=$(curl -sSL 'https://api.github.com/users/${{ inputs.bot-name }}' | jq .id)
```

Comment:
We should only do this if we are deriving the email address because none was provided.